commands.go

```diff
@@ -2472,6 +2473,18 @@ func (c cmdable) ZInterCard(ctx context.Context, limit int64, keys ...string) *I
 	_ = c(ctx, cmd)
 	return cmd
 }
+func (c cmdable) ZMPop(ctx context.Context, order string, count int64, keys ...string) *ZArrayWithKeyCmd {
+	args := make([]interface{}, 2+len(keys), 5+len(keys))
+	args[0] = "zmpop"
+	args[1] = len(keys)
+	for i, key := range keys {
+		args[2+i] = key
+	}
+	args = append(args, strings.ToLower(order), "count", count)
```

Comment:
@chayim 

What is the policy about wrong `order` provided? Should we check against correct values (e.g `min`, `max` ) or just rely on redis server error response and assume that user familiar with documentation?

Reply:
Clients generally stick to order provided by the redis commands.

Reply:
So is there something I should do separately about this part? @chayim @vladvildanov

Reply:
@SoulPancake @chayim Maybe it was a little misunderstanding, I'm just wondering should we check value of `string order` argument on the client side or it's okay if server exception will be thrown in case if user provide wrong value? Since, `string order` is not an enum (google says Go doesn't support enums), so we never know which value user will provide